src/CistromeHGW.js

```diff
@@ -139,10 +161,20 @@ export default function CistromeHGW(props) {
         );
     }, [viewConfig]);
 
+    /*
+     * Add SVG icons.
+     */
+    useEffect(() => {
```

Comment:
I think it would be better to remove this code from the CistromeHGW component. I think we do not even need to use D3 to render these icons, we can just rely on plain react, and render `<path d={SEARCH.path} />` within the svg elements for example. And the track controls will probably become more complex, especially with the search functionality, so we should move them to their own component that takes `onSortAscClick` and `onSortDescClick` as props.